test-case-reporting/test/test_result_record.test.ts

```diff
@@ -36,6 +36,10 @@ jest.mock('../src/db', () => ({
       client: 'sqlite3',
       connection: ':memory:',
       useNullAsDefault: true,
+      pool: {
+        afterCreate: (connection: any, callback: any): any =>
```

Comment:
Nice work figuring this out! Let's add comment explaining what's going on

Reply:
Done. I added another comment as well, PTAL.